testcharms/charm-hub/charms/juju-qa-test/src/charm.py

```diff
@@ -54,27 +57,28 @@ def _on_update_status(self, _):
         details = "it is now: {0}".format(date_time)
         self.unit.status = ActiveStatus(details)
 
-    def _check_foo_file_config(self):
+    def _check_foo_file_config(self) -> bool:
```

Comment:
I'd prefer if referred to `foo-file` as just `test-config` or something but that's no big deal.
```suggestion
    def _is_foo_file_config_changed(self) -> bool:
```